odoo/models.py

```diff
@@ -66,10 +66,9 @@
 from .tools.lru import LRU
 
 _logger = logging.getLogger(__name__)
-_schema = logging.getLogger(__name__ + '.schema')
 _unlink = logging.getLogger(__name__ + '.unlink')
 
-regex_order = re.compile('^(\s*([a-z0-9:_]+|"[a-z0-9:_]+")(\s+(desc|asc))?\s*(,|$))+(?<!,)$', re.I)
+regex_order = re.compile(r'^(\s*([a-z0-9:_]+|"[a-z0-9:_]+")(\s+(desc|asc))?\s*(,|$))+(?<!,)$', re.I)
```

Comment:
Why? Lint on the subject in your editor?

Reply:
Yep, warning from pylint and also the `\` is confusing without `r"`

https://pycodequ.al/docs/pylint-messages/w1401-anomalous-backslash-in-string.html